src/torchio/visualization.py

```diff
@@ -102,6 +104,9 @@ def plot_volume(
     axi_axis.set_title('Axial')
 
     plt.tight_layout()
+    if figtitle is not None:
+        plt.suptitle(figtitle, y=figtitle_y, fontsize=15)
```

Comment:
Instead of hard-coding the font size, can you remove it or add a kwarg such as `title_font_size`?